pokedex/src/components/filter/index.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import iconFilter from '../../assets/arrow-down.svg';
+import styles from './styles.module.scss';
+
+function Filter({ onClick, value }) {
+  return (
+    <div onClick={onClick} className={styles.containFilter}>
+      {value ? (
+        <a className={styles.filter}>A-Z</a>
+      ) : (
+        <a className={styles.filter}>#</a>
+      )}
```

Comment:
Puedes usar String Interpolation para hacerlo mas legible 

```suggestion
        <a className={styles.filter}>{`${value ? 'A-Z' : '#'}`}</a>
```

Reply:
Actualizado